packages/editor-ui/src/components/OutputPanel.vue

```diff
@@ -277,6 +283,14 @@ watch(defaultOutputMode, (newValue: OutputType, oldValue: OutputType) => {
 const activatePane = () => {
 	emit('activatePane');
 };
+
+const shouldShowWarning = computed(() => {
```

Comment:
Can I suggest a more specific variable name since `OutputPanel` is a generic component? Something like `noAiToolsUsed` or similar